Web/utils.py

```diff
@@ -58,6 +59,7 @@ def create(self):
         logger.info('SlotMachienPC pid: %d' % self.process.pid)
         self.stopped = False
         self.last_status = self.process.stdout.readline()
+        self.last_door_status = self.process.stdout.readline()
```

Comment:
I think this might break things, since the code for the process you're reading from isn't modified

Reply:
yeah I didn't quite understand why you would read from stdout, I think my newest change should fix this issue.